lib/cartopy/tests/crs/test_spilhaus.py

```diff
@@ -0,0 +1,40 @@
+# Copyright Crown and Cartopy Contributors
+#
+# This file is part of Cartopy and is released under the BSD 3-clause license.
+# See LICENSE in the root of the repository for full licensing details.
+"""
+Tests for the HEALPix projection.
+"""
+from packaging.version import parse as parse_version
+import pyproj
+import pytest
+
+import cartopy.crs as ccrs
+from .helpers import check_proj_params
+
+
+proj_version = parse_version(pyproj.proj_version_str)
+common_arg = {
+    'datum=WGS84',
+    'ellps=WGS84',
+    'no_defs',
+    'units=m',
+}
+@pytest.mark.skipif(
+    (proj_version < parse_version("9.6.0")),
+    reason="Requires pyproj > 3.7.0"
```

Comment:
I think you could add this mark at the module level since all of these tests require PROJ>9.6.
https://docs.pytest.org/en/stable/example/markers.html

The text of the reason needs to be updated to PROJ specific in all of the reasons.
```suggestion
    reason="Requires PROJ >= 9.6.0"
```